doc/source/pbm-commands.rst

```diff
@@ -41,7 +68,41 @@ The command accepts the following flags:
      - Upload the config information from a YAML file
    * - ``--set=SET``
      - Set a new config option value. Specify the option in the <key.name=value> format.
+   * - ``-o``, ``--out=text``
+     - Shows the output format as either plain text or a JSON object. Supported values: text, json
    
+.. container:: toggle
+
+   .. container:: header
+
+      **JSON output**
+
+   .. code-block:: javascript
+
+      {
+        "pitr": {
+          "enabled": false,
+          "oplogSpanMin": 0
+        },
+        "storage": {
+          "type": "filesystem",
+          "s3": {
+            "region": "",
+            "endpointUrl": "",
+            "bucket": ""
+          },
+          "azure": {},
+          "filesystem": {
+            "path": "<my-backup-dir>"
+          }
+        },
+        "restore": {
+          "batchSize": 500,
+          "numInsertionWorkers": 10
+        },
+        "backup": {}
+      }
+
```

Comment:
`pbm config --force-resync` also has json output:
```json
{"msg":"Storage resync started"}
```

Reply:
`pbm config --set=` also has json output reflecting the changed value. For example:
```
$ pbm config --set=pitr.enabled=false
[{"key":"pitr.enabled","value":"false"}]
```